deep4cast/inferences/gradient_based.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is not needed?

Reply:
Also, what is the semantics of the `inferences` folder?

Reply:
Removing it.